jgt_tools/docs/build_docs.py

```diff
@@ -64,16 +64,26 @@ def _build_conf():
 def build():
     """Build the docs."""
     # Setup environment variables
-    commit_id = subprocess.check_output(
-        ["git", "rev-parse", "HEAD"], cwd=BASE_DIR, universal_newlines=True
-    )
+    try:
+        commit_id = subprocess.check_output(
+            ["git", "rev-parse", "HEAD"],
+            cwd=BASE_DIR,
+            universal_newlines=True,
+            stderr=subprocess.DEVNULL,
```

Comment:
Without adding `stderr=subprocess.DEVNULL`, the output from the `git rev-parse HEAD` was getting printed on the command line, which wasn't terribly useful. Same for below.